tests/test_integration_fast_train.py

```diff
@@ -123,6 +123,7 @@ def test_train_timing(self):
                     range_x=np.pi / 4,
                     mode=("bilinear", "nearest"),
                     align_corners=True,
+                    dtype=np.float64,
```

Comment:
@Nic-Ma  if we use np.float32 here might speed up it a bit...

Reply:
OK, let's mark it here for future performance tuning.

Thanks.